schema/variable_schema.go

```diff
@@ -3,16 +3,33 @@ package schema
 import (
 	"github.com/hashicorp/hcl-lang/lang"
 	"github.com/hashicorp/hcl-lang/schema"
+	"github.com/hashicorp/terraform-schema/internal/schema/refscope"
 	"github.com/hashicorp/terraform-schema/module"
 	"github.com/zclconf/go-cty/cty"
 )
 
-func SchemaForVariables(vars map[string]module.Variable) (*schema.BodySchema, error) {
+func SchemaForVariables(vars map[string]module.Variable, path string) (*schema.BodySchema, error) {
```

Comment:
```suggestion
func SchemaForVariables(vars map[string]module.Variable, modPath string) (*schema.BodySchema, error) {
```
Nitpick - just to clarify what path this is.